google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseAdminClientImplTest.java

```diff
@@ -246,6 +250,27 @@ public void updateDatabaseDdlOpAlreadyExists() throws Exception {
     assertThat(op.getName()).isEqualTo(originalOpName);
   }
 
+  @Test
```

Comment:
We could add more end-to-end tests to assert the exception we get when we delete the database (after enabling the field). Then, disable the field and try to delete it.